instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/InstrumenterBuilder.java

```diff
@@ -110,6 +110,15 @@
     return newInstrumenter(InstrumenterConstructor.internal(), SpanKindExtractor.alwaysInternal());
   }
 
+  /**
+   * Returns a new {@link Instrumenter} which will create spans with kind determined by the passed
+   * {@code spanKindExtractor} and do no context propagation.
+   */
+  public Instrumenter<REQUEST, RESPONSE> newInstrumenter(
```

Comment:
would `INPUTS, OUTPUTS` make more sense here, rather than request/response?

Reply:
It would, especially for non-HTTP instrumenters. @anuraaga WDYT?

Reply:
I feel like request / response are fairly standard terms even in a general sense, even for DBs for example. INPUTS OUTPUTS just seems a bit too generic, I think the difference vs request / response is the latter implies being "remote" which is what we care about when instrumenting.

Reply:
Hmm, let's try a few more conventions first (RPC, messaging) - we can always revisit this topic a bit later.
In the meantime I'm going to merge this PR and take on the next semconv attribute group.